nobrainer/models/autoencoder.py

```diff
@@ -0,0 +1,82 @@
+"""Model definition for Autoencoder.
+"""
+
+import tensorflow as tf
+from tensorflow.keras import layers, models
+import math
```

Comment:
Can you `import math` before tensorflow? `math` is part of the standard library, so it's good to put it above 3rd party libraries.